_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+
+Thirdly, an enabling team need not commit all of their time to being an enabling team. For example, in the case of the IT Strategy team they are also responsible for setting the IT vision for the department, and thus cannot dedicate 100%(-20%) of their time to their enabling function. In this particular case, for example, we recommend that the IT Strategy spend a maximum of (100-20)/2 percent of their time on enabling other teams.
+
+##### External Interactions
+
+This is an initial proposal and is intended to allow for iterations to refine and modify the process as needed.
+
+###### Interactions
+
+The book Team Topologies, referenced below, recommends that Enabling Teams work with other teams for a period (potentially a few months) [TO-DO: Quote needed] in order to assist them with onboarding new technologies and approaches, and improving their existing processes and procedures. The goal is to ensure the interactions are brief enough that the teams being enabled do not become dependent on the enabling teams. The process improvements should stand on their own, and should not involve [toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/). That is to say, the new processes and procedures should aim to automate or improve the existing processes, not lend temporary person-power to the teams. Given the short and iterative nature of the interactions between the teams (duration of interactions should be defined by number of sprints) the enabling teams must identify quick wins -- the amount of automation that could be done within the limited time frame that would result in the greatest amount of automated work.
+
+###### Tooling
+
+[TO-DO: Quote from Team Topologies regarding varying levels of team communication]
+
+The amount of collaboration between the enabling teams and the enabled teams should vary significantly during the time which the enabling team is paired with the enabled team. As such, collaboration tools that allow for the rapid creation and disintegration should be leveraged. An example of a viable tool for such a requirement would be Slack, where groups can be created and disposed of with relative ease.
+
+#### Process
+
+##### Teams Iterative Process
+
+- All teams within the directorate have synchronized sprints
+- Representatives from each enabling team within the enabling directorate meet prior to the synchronized sprint planning for the director to set directorate-level priorities
+- During the synchronized sprint planning meetings, ensure directorate-level priorities are broken down into manageable tasks
+
+##### Engaging the Enabling Teams
+
+- Potential teams to be enabled should contact the enabling teams
+- An item is created on the kanban board of the enabling team
```

Comment:
Maybe worthwhile to have a single kanban board for the whole directorate. A single "enabling" kanban board so that WIP and backlog is visible to the director in one place and has an easy monitoring ability to identify work that has been "in progress" for too long.

This is similar to the lean model where an hypothesis is produced and the validation of that hypothesis is not expected to take months otherwise it may indicate the hypothesis is false.

Reply:
This may be hard to manage but we could attempt it. Roll up of the cards may be a better option (not sure if there are solutions for this already).

Reply:
Only thing I would caution is about redundant work. Need to avoid creating multiple "boards" to track the same things twice or more.

Reply:
This is the idea of the synchronized sprints. 

Director meets with representative of teams
Teams discuss who will do what
Teams all have their sprint planning
Teams prioritize the work that they have taken on as part of the meeting with the director
Entire directorate completes sprint
Repeat (what did we all get done, what remains to be done)

So perhaps some master board with large tasks you can link to? Or perhaps milestones, if we all have our own projects or some such thing? I see the use case attempting to be solved here ... just not sure what tools would fit this requirement ... it's a neat idea though. 'sub kanban boards' 

Granted, that is supposed to be the responsibility of the product owner. Ensure they understand the greater priorities and goals and ensure that they're not doing double work and so forth. So the nested kanban board approach sounds like another way of solving the same problem the meetings with the director is supposed to solve (coordinating work between the teams in the directorate)